instrumentation/all/README.md

```diff
@@ -1,6 +1,6 @@
-# opentelemetry-adapters-all
+# opentelemetry-instrumentation-all
 
-The `opentelemetry-adapters-all` gem is an all-in-one distribution of community maintained instrumentation adapters. Instrumentation adapters are packaged as individual gems for flexibility and maintainability. Instead of having to require each adapter individually, applications can depend on this all-in-one gem as a convenient alternative.
+The `opentelemetry-instrumentation-all` gem is an all-in-one distribution of community maintained instrumentation instrumentations. Instrumentation instrumentations are packaged as individual gems for flexibility and maintainability. Instead of having to require each instrumentation individually, applications can depend on this all-in-one gem as a convenient alternative.
```

Comment:
```suggestion
The `opentelemetry-instrumentation-all` gem is an all-in-one distribution of community maintained instrumentation. Instrumentation libraries are packaged as individual gems for flexibility and maintainability. Instead of having to require each instrumentation individually, applications can depend on this all-in-one gem as a convenient alternative.
```